src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -658,9 +658,9 @@ public override async Task<VSInternalWrapWithTagResponse> RazorWrapWithTagAsync(
             {
                 var csharpRequestParams = new FoldingRangeParams()
                 {
-                    TextDocument = new()
+                    TextDocument = new VSTextDocumentIdentifier()
                     {
-                        Uri = csharpSnapshot.Uri
+                        Uri = csharpSnapshot.Uri,
                     }
```

Comment:
Should this modify the uri, so we don't lose project context (or other data)?

Reply:
Fixed. I'll do a follow up PR to think about making this easier to do because right now it's a bit messy and easy to get wrong